packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { COLLABORATION_HEARTBEAT_INTERVAL } from '@/constants';
+import { useUsersStore } from '@/stores/users.store';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useCollaborationStore } from '@/stores/collaboration.store';
+import { onBeforeUnmount } from 'vue';
+import { onMounted } from 'vue';
+import { computed, ref } from 'vue';
+
+const collaborationStore = useCollaborationStore();
+const usersStore = useUsersStore();
+const workflowsStore = useWorkflowsStore();
+
+const heartbeatInterval = COLLABORATION_HEARTBEAT_INTERVAL;
+const heartbeatTimer = ref(null as null | NodeJS.Timeout);
```

Comment:
```suggestion
const heartbeatTimer = ref<NodeJS.Timeout | null>(null as null | NodeJS.Timeout);
```

Tho I don't think we should rely on NodeJS typings on the frontend. I think it could as well be a `number`